data-reporting-service/investigation-service/src/test/java/gov/cdc/etldatapipeline/investigation/TestUtils.java

```diff
@@ -0,0 +1,20 @@
+package gov.cdc.etldatapipeline.investigation;
+
+import org.apache.commons.io.FileUtils;
+import org.springframework.core.io.ClassPathResource;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+public class TestUtils {
```

Comment:
We can consider moving this to the newly created Common Utils module and reference it from there.